lua/telekasten.lua

```diff
@@ -2487,6 +2512,8 @@ local function Setup(cfg)
         -- merge everything but calendar opts
         -- they will be merged later
         if k ~= "calendar_opts" then
+            if k == "home" then
+                v = CleanPath(v)
```

Comment:
There seems to be no `end` for the if. Are you sure you tested this? Or is GitHub playing tricks on me?

Reply:
I looked, and what happened was that I had fixed it on my laptop, but had not committed and pushed to GitHub. Got onto my work computer and did a pull request and didn't think to check. This is my first time doing a PR so I got a bit excited and didn't think too clearly. I've fixed it though!